inst/sql/sql_server/concept_plausible_unit_concept_ids.sql

```diff
@@ -39,13 +39,8 @@ FROM
     		AND c.cohort_definition_id = @cohortDefinitionId
     	}
 		WHERE m.@cdmFieldName = @conceptId
-		  AND {@plausibleUnitConceptIds == '' | @plausibleUnitConceptIds == 'NA'}?{
-		  m.unit_concept_id IS NOT NULL
-		  }:{
-		  m.unit_concept_id NOT IN (@plausibleUnitConceptIds)
-		  }
+		  AND COALESCE (m.unit_concept_id, -1) NOT IN (@plausibleUnitConceptIds) -- '-1' stands for the cases when unit_concept_id is null
```

Comment:
`plausibleUnitConceptIds` will not be empty anymore?

Reply:
yes, to reflect NULL in the data, I use '-1' in @plausibleUnitConceptIds. and it will be then compared with COALESCE (m.unit_concept_id, -1)